src/main/java/frc/robot/commands/FinishClimb.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.subsystems.Climber;
+
+public class FinishClimb extends CommandBase {
```

Comment:
Finish climb is just running arm1 up and arm2 down at the same time.

Rename this class to "AutoClimb" and create a "finishClimb" method in the climber class